### DISCUSSION TITLE
Master Deep Voice Cloning in Minutes: Unleash Your Vocal Superpowers! Free and Locally on Your PC

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [152334H/DL-Art-School#72] [Show and tell]
I made a pull request too please accept if possible : https://github.com/152334H/DL-Art-School/pull/71

[**Master Deep Voice Cloning in Minutes: Unleash Your Vocal Superpowers! Free and Locally on Your PC**](https://youtu.be/OiMRlqcgDL0)

[![image](https://github.com/152334H/DL-Art-School/assets/19240467/1a5b1d93-7fc8-473b-b88a-d44beee105a9)
](https://youtu.be/OiMRlqcgDL0)

This tutorial is based on

Ozen Toolkit for data preprocessing
DLAS for Training
Tortoise TTS Fast for speech synthesis

**Comment by vitalifa** [comment]
I've seen your name appear in a few places over github over the past months, so I know you have probably been asking the same questions a lot of us are.

This video managed to capture so much of the information floating around across issues and repos and bring it all together. Highly valuable to this repo.

TYSM :metal:

**Reply by FurkanGozukara** [reply]
> I've seen your name appear in a few places over github over the past months, so I know you have probably been asking the same questions a lot of us are.
> 
> This video managed to capture so much of the information floating around across issues and repos and bring it all together. Highly valuable to this repo.
> 
> TYSM 🤘

thank you so much for the amazing comment
